HISTORY.md

```diff
@@ -0,0 +1,255 @@
+## [v2.2.0](https://github.com/voxpupuli/puppet-staging/tree/v2.2.0) (2017-02-11)
```

Comment:
The HISTORY.md is supposed to contain what *was* in CHANGELOG.md before running gcg...